api/backend.go

```diff
@@ -15,22 +16,20 @@ import (
 
 // StatusBackend defines the contract for the Status.im service
 type StatusBackend interface {
-	// IsNodeRunning() bool                       // NOTE: Only used in tests
 	StartNode(config *params.NodeConfig) error // NOTE: Only used in canary
 	StartNodeWithKey(acc multiaccounts.Account, password string, keyHex string, conf *params.NodeConfig) error
 	StartNodeWithAccount(acc multiaccounts.Account, password string, conf *params.NodeConfig) error
 	StartNodeWithAccountAndInitialConfig(account multiaccounts.Account, password string, settings settings.Settings, conf *params.NodeConfig, subaccs []*accounts.Account) error
 	StopNode() error
-	// RestartNode() error // NOTE: Only used in tests
 
 	GetNodeConfig() (*params.NodeConfig, error)
 	UpdateRootDataDir(datadir string)
 
-	// SelectAccount(loginParams account.LoginParams) error
+	SelectAccount(loginParams account.LoginParams) error
```

Comment:
Just wondering... if this function calls `func (m *DefaultManager) SelectAccount(loginParams LoginParams) error` and it expects `loginParams.Password` to be provided, won't that return an error in case of a keycard migrated profile?

Reply:
This change only reintroduces the function signature to the `StatusBackend` interface. The implementation details of this function (whether `loginParams.Password` is expected etc) are a concern for the implementing struct. 

However existing code, unaltered by this PR, that implements this function signature already uses the `loginParams.Password` property.

```go
func (m *DefaultManager) SelectAccount(loginParams LoginParams) error {
	m.mu.Lock()
	defer m.mu.Unlock()

	m.accountsGenerator.Reset()

	selectedChatAccount, err := m.unlockExtendedKey(loginParams.ChatAddress.String(), loginParams.Password)
	if err != nil {
		return err
	}
	m.watchAddresses = loginParams.WatchAddresses
	m.mainAccountAddress = loginParams.MainAccount
	m.selectedChatAccount = selectedChatAccount
	return nil
}
```

This code is called via the `LoginAccount` API method. So perhaps a different API method is called when logging in via Keycard, because the failure would already be an issue without the changes in this PR.

Reply:
@Samyoul yes, I see what you've changed, no problem with your change at all, I just used the opportunity to ask does that somehow affect the keycard users, nothing else, sorry about confusion. :)